[2025-03-09T16:59:55.728+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-09T16:57:40.995662+00:00 [queued]>
[2025-03-09T16:59:55.747+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-09T16:57:40.995662+00:00 [queued]>
[2025-03-09T16:59:55.747+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-03-09T16:59:55.771+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): data_storage_task> on 2025-03-09 16:57:40.995662+00:00
[2025-03-09T16:59:55.778+0000] {standard_task_runner.py:57} INFO - Started process 11734 to run task
[2025-03-09T16:59:55.783+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'customer_churn_ml_pipeline', 'data_storage_task', 'manual__2025-03-09T16:57:40.995662+00:00', '--job-id', '400', '--raw', '--subdir', 'DAGS_FOLDER/customer_churn.py', '--cfg-path', '/tmp/tmp4rgn9ska']
[2025-03-09T16:59:55.787+0000] {standard_task_runner.py:85} INFO - Job 400: Subtask data_storage_task
[2025-03-09T16:59:55.818+0000] {logging_mixin.py:154} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-03-09T16:59:55.886+0000] {task_command.py:416} INFO - Running <TaskInstance: customer_churn_ml_pipeline.data_storage_task manual__2025-03-09T16:57:40.995662+00:00 [running]> on host 157237bff8f7
[2025-03-09T16:59:56.074+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='customer_churn_ml_pipeline' AIRFLOW_CTX_TASK_ID='data_storage_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-09T16:57:40.995662+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-09T16:57:40.995662+00:00'
[2025-03-09T16:59:56.076+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-09T16:59:56.078+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n\t\t\tcd /opt/***\n                        dvc pull /opt/***/data/transformed\n                        python /opt/***/executables/data_storage.py && \t\t\tgit add logs &&    \t\t\tgit status &&                         git commit -m "Updated stored data version" -a &&                         dvc push && git push\n                    ']
[2025-03-09T16:59:56.090+0000] {subprocess.py:86} INFO - Output:
[2025-03-09T17:00:01.495+0000] {subprocess.py:93} INFO - Everything is up to date.
[2025-03-09T17:00:04.685+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,685 : INFO : Starting data storage for customer_data.csv
[2025-03-09T17:00:04.686+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,685 : INFO : Connecting to Database
[2025-03-09T17:00:04.723+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,723 : INFO : Connecting to Database Successfull
[2025-03-09T17:00:04.724+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,723 : INFO : Reading insert script template
[2025-03-09T17:00:04.725+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,724 : INFO : Executing script to Database
[2025-03-09T17:00:04.725+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,724 : INFO : Inserting data to database
[2025-03-09T17:00:04.919+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,919 : ERROR : Error storing data: ('42000', '[42000] [Microsoft][ODBC Driver 17 for SQL Server][SQL Server]The incoming tabular data stream (TDS) remote procedure call (RPC) protocol stream is incorrect. Parameter 6 (""): The supplied value is not a valid instance of data type float. Check the source data for invalid values. An example of an invalid value is data of numeric type with scale greater than precision. (8023) (SQLExecDirectW)')
[2025-03-09T17:00:04.924+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,924 : INFO : Starting data storage for database_data.csv
[2025-03-09T17:00:04.925+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,925 : INFO : Connecting to Database
[2025-03-09T17:00:04.951+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,950 : INFO : Connecting to Database Successfull
[2025-03-09T17:00:04.951+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,951 : INFO : Reading insert script template
[2025-03-09T17:00:04.952+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,951 : INFO : Executing script to Database
[2025-03-09T17:00:04.952+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:04,951 : INFO : Inserting data to database
[2025-03-09T17:00:13.592+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:13,592 : INFO : Executed script successfully !!
[2025-03-09T17:00:13.593+0000] {subprocess.py:93} INFO - 2025-03-09 17:00:13,593 : INFO : Connection closed !!
[2025-03-09T17:00:13.899+0000] {subprocess.py:93} INFO - On branch main
[2025-03-09T17:00:13.900+0000] {subprocess.py:93} INFO - Your branch is up to date with 'origin/main'.
[2025-03-09T17:00:13.900+0000] {subprocess.py:93} INFO - 
[2025-03-09T17:00:13.900+0000] {subprocess.py:93} INFO - Changes to be committed:
[2025-03-09T17:00:13.901+0000] {subprocess.py:93} INFO -   (use "git restore --staged <file>..." to unstage)
[2025-03-09T17:00:13.901+0000] {subprocess.py:93} INFO - 	new file:   logs/dag_id=customer_churn_ml_pipeline/run_id=manual__2025-03-09T16:57:40.995662+00:00/task_id=data_storage_task/attempt=1.log
[2025-03-09T17:00:13.901+0000] {subprocess.py:93} INFO - 	modified:   logs/dag_id=customer_churn_ml_pipeline/run_id=manual__2025-03-09T16:57:40.995662+00:00/task_id=data_transformation_task/attempt=1.log
[2025-03-09T17:00:13.902+0000] {subprocess.py:93} INFO - 	modified:   logs/dag_processor_manager/dag_processor_manager.log
[2025-03-09T17:00:13.902+0000] {subprocess.py:93} INFO - 	modified:   logs/data_storage.log
[2025-03-09T17:00:13.902+0000] {subprocess.py:93} INFO - 	modified:   logs/scheduler/2025-03-09/customer_churn.py.log
[2025-03-09T17:00:13.902+0000] {subprocess.py:93} INFO - 
[2025-03-09T17:00:13.903+0000] {subprocess.py:93} INFO - Untracked files:
[2025-03-09T17:00:13.903+0000] {subprocess.py:93} INFO -   (use "git add <file>..." to include in what will be committed)
[2025-03-09T17:00:13.903+0000] {subprocess.py:93} INFO - 	customer_churn_stats/feature_repo/data/online_store.db
[2025-03-09T17:00:13.903+0000] {subprocess.py:93} INFO - 	customer_churn_stats/feature_repo/data/registry.db
[2025-03-09T17:00:13.904+0000] {subprocess.py:93} INFO - 	mlartifacts/
[2025-03-09T17:00:13.904+0000] {subprocess.py:93} INFO - 	models/
[2025-03-09T17:00:13.904+0000] {subprocess.py:93} INFO - 	online.db
[2025-03-09T17:00:13.905+0000] {subprocess.py:93} INFO - 
